base-notebook/start.sh

```diff
@@ -27,14 +27,23 @@ if [ $(id -u) == 0 ] ; then
     fi
 
     # Enable sudo if requested
-    if [ ! -z "$GRANT_SUDO" ]; then
+    if [ "$GRANT_SUDO" == "1" ]; then
```

Comment:
All of the READMEs give `-e GRANT_SUDO='yes'` as an example. Should we keep that or update all the documentation? I'd imagine supporting `1` and only `1` going forward might break a few users.